$MirOS$

	This should not happen, but it does. Also, it should be handled
	more nicely. But hey, this about matches the overall code quality.

--- vm/nmunix_gui.cpp.orig	Fri Apr 11 00:27:24 2008
+++ vm/nmunix_gui.cpp	Thu Apr 19 13:16:53 2012
@@ -620,6 +620,8 @@ bool equals(graphics_font &other)
 	graphics_font(WObject font){
 		if (font != 0){
 			char *n = stringToNativeText(WOBJ_FontName(font));
+			if (!n)
+				asprintf(&n, "UnnamedFont%08X", (unsigned)font);
 			name.str = copyString(n);
 			name.len = strlen(name.str);
 			free(n);
